recipe/bld.bat

```diff
@@ -29,3 +29,7 @@ if errorlevel 1 exit 1
 :: Test.
 ctest -C Release
 if errorlevel 1 exit 1
+
+:: Workaround for package that got build with latet version that renamed these.
+copy %LIBRARY_BIN%\\libexpat.dll %LIBRARY_PREFIX%\\expat.dll || exit 1
```

Comment:
This should be copying into %LIBRARY_BIN% agai, not %LIBRARY_PREFIX%

Reply:
Jeez! I fixed one and forgot about the other. This is what I get for sending PR when I should be resting. Thanks!